README.md

```diff
@@ -52,4 +52,4 @@ As my Bitwarden instance is a critical part of my daily workflow and part of acc
 
 Your Bitwarden instance will go to sleep after 30 minutes of no activity. This should not be too bad of an issue due to the fact that you can maintain a local copy. However if you are adding, you may wish to have a cron job which polls your instance to keep it avaliable (read: Pingdom set to 15 minute intervals or any website status checker).
 
-The JawsDB instance comes with 5MB of storage space. I found this sufficient enough for my own personal backups even with 700+ entries, two orgs, and 4 members. You may find if you are attaching content, that you might exceed this but I suggest attach files in base64 encoded content to preserve portability.
\ No newline at end of file
+The JawsDB instance comes with 5MB of storage space. I found this sufficient enough for my own personal backups even with 700+ entries, two orgs, and 4 members. You may find if you are attaching content, that you might exceed this but I suggest attach files in base64 encoded content to preserve portability.
```

Comment:
I don't see any changes here? Could you clarify the correction here?

Reply:
The change is on line 17 (simply -> simple). I'm not sure why Git is highlighting line 55 instead, but take a closer look at the changes view and you'll be able to see it :)

Reply:
I'm not sure why Git is highlighting that line, but indeed, the change is not there :)